aggregate/__main__.py

```diff
@@ -149,20 +165,67 @@ def main():
         project=config_values['PROJECT'],
     )
 
+    images = {}
     for function in functions:
-        # Create the function and it's corresponding pubsub and subscription.
-        fxn, _, _ = create_cloud_function(
-            name=function,
-            config_values=config_values,
-            service_account=config_values['GCP_SERVICE_ACCOUNT'],
-            pubsub_topic=pubsub,
-            cloud_service=cloud_service,
-            function_bucket=function_bucket,
-            source_archive_object=source_archive_object,
-            slack_channel=slack_channel,
-        )
+        if config_values.get('TYPE') == 'function':
```

Comment:
Can we convert all the functions easily, so we don't need these two cases?

Reply:
We can, but for functions that currently work I wanted to leave them unaffected. Sometimes a Cloud Run job will be overkill. So the option for either seems like it would be useful.

Reply:
\+1 to moving all functions to cloud run, should make it slightly easier to track all the pieces that operate on data.